cmd/geth/main.go

```diff
@@ -364,11 +364,12 @@ func execJSFiles(ctx *cli.Context) {
 func unlockAccount(ctx *cli.Context, am *accounts.Manager, account string) (passphrase string) {
 	var err error
 	// Load startup keys. XXX we are going to need a different format
-	// Attempt to unlock the account
-	passphrase = getPassPhrase(ctx, "", false)
+
 	if len(account) == 0 {
 		utils.Fatalf("Invalid account address '%s'", account)
 	}
+	// Attempt to unlock the account
+	passphrase = getPassPhrase(ctx, "Unlocking account "+account, false)
```

Comment:
I think we should abbreviate addresses for display, especially in the prompt.
And maybe match them by prefix so less stuff needs to be typed.